cardano-testnet/src/Testnet/Util/Runtime.hs

```diff
@@ -67,16 +72,16 @@ data NodeRuntime = NodeRuntime
 
 data PoolNode = PoolNode
   { poolRuntime :: NodeRuntime
-  , poolKeys :: PoolNodeKeys
+  , poolKeys :: PoolNodeKeys -- used in Alonzo LeadershipShedule
```

Comment:
I'm curious about the comment.  We shouldn't have to care where this data is used.  For example new tests might be added that make use of this.

Reply:
I'm about to change the code so I naturally need to know where something is defined and where it is used.
I think I also found stuff that is defined but not not used.
IMHO most of the time, it is not a good idea to add record fields for data that is not used.
Just because the data might be used in the future is not a good reason.